tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -87,6 +87,7 @@ namespace Internal {
 namespace RDF {
 class GraphCreatorHelper;
 void ChangeEmptyEntryRange(const ROOT::RDF::RNode &node, std::pair<ULong64_t, ULong64_t> &&newRange);
+void ChangeBeginAndEndEntries(const RNode &node, ULong64_t start, ULong64_t end);
```

Comment:
The data members are `fBeginEntry` and `fEndEntry` so let's change the name of the function parameter for symmetry
```suggestion
void ChangeBeginAndEndEntries(const RNode &node, ULong64_t begin, ULong64_t end);
```